### PR TITLE
Updated README.md (#18)

### PR DESCRIPTION
**Pull request by rmasuoka**
- Remove "-doc" in the links in the text
- Close links with ")" in the table
- Make the ATT&CK license notice two lines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,36 +10,36 @@ You can find the resulting CSV files for the different versions of ATT&CK in the
...
+|v5.1|[o](attack-csv-data/v5.1/)|
+|v5.0|[o](attack-csv-data/v5.0/)|
+|v4.0|[o](attack-csv-data/v4.0/)|
+|v3.0|[o](attack-csv-data/v3.0/)|
+|v2.0|[o](attack-csv-data/v2.0/)|
+|v1.0|[o](attack-csv-data/v1.0/)|
 
 
-This project makes use of ATT&CK® 
+This project makes use of ATT&CK®
```
Comment (stmtstk) [inline README.md:44]: Please remove the space at the end of the sentence

Comment (rmasuoka) [reply README.md:44]: @stmtstk Those two spaces to give a line break. (I followed the APS page example - https://chrome.google.com/webstore/detail/attck-powered-suit/gfhomppaadldngjnmbefmmiokgefjddd)

**Comment by rmasuoka** [comment]
I will also change the link names to the folder names as @shiracamus has suggested.

**Comment by rmasuoka** [comment]
I have made the changes to the link names in the table and I believe this PR is ready to go.

File: README.md (hunk 1 above)
Comment (stmtstk) [reply README.md:44]: Sorry. You use the right notation.

**Review by stmtstk — APPROVED** [review]

**Pull request merged**